role/db.go

```diff
@@ -103,3 +105,18 @@ func (s *Store) deleteRolePermission(ctx context.Context, r Role) error {
 	}
 	return nil
 }
+
+func (s *Store) deleteRoleUser(ctx context.Context, r Role) error {
```

Comment:
NITPICK: IMO `deleteRoleUser` (and `deleteRolePermission`) leak an internal abstraction (the fact that there exists a `role_member` table) and are a bit confusing ("what is a role-user?" was my first thought). What do you think about `removeRoleFromUser` / `removePermissionFromRole`?

Reply:
I can rename them, that's fine

Reply:
Fixed